src/frontend/screens/AppPasscode/InputPasscode.tsx

```diff
@@ -54,7 +54,7 @@ export const InputPasscode = ({
   const inputRef = useBlurOnFulfill({
     value: inputValue,
     cellCount: CELL_COUNT,
-  });
+  }) as React.RefObject<TextInput>;
```

Comment:
would it make sense to update the `PasscodeInput` component type declaration to avoid needing the type coercion here? Maybe not worth doing...

```diff
-export const PasscodeInput = React.forwardRef<TextInput, PasscodeInputProps>(
+export const PasscodeInput = React.forwardRef<TextInput | null, PasscodeInputProps>(
```